### PR TITLE
Fix for overlapping tiles with long image names that aren't truncating

### PR DESCRIPTION
**Pull request by sg00dwin**
Fixes https://github.com/openshift/origin-web-console/issues/1003

screen shot 2016-12-20 at 3 52 00 pm

screen shot 2016-12-20 at 4 36 16 pm
screen shot 2016-12-20 at 4 23 09 pm
screen shot 2016-12-20 at 4 21 03 pm

### CODE REVIEW

**Comment by sg00dwin** [comment]
@spadgett PTAL thx

File: app/views/overview/_image-names.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<span>{{podTemplate.spec.containers[0].image | imageStreamName}}</span>
+<span class="truncate">{{podTemplate.spec.containers[0].image | imageStreamName}}</span>
 <span ng-repeat="id in imageIDs" title="{{id}}">
```
Comment (spadgett) [inline app/views/overview/_image-names.html]: Should we truncate the image IDs as well?

**Comment by spadgett** [comment]
[merge]

**Comment by spadgett** [comment]
[merge]

**Pull request merged**